index.js

```diff
@@ -69,14 +70,15 @@ module.exports = {
                 return all;
             }.bind(this), [])
         ).then(function(contexts) {
-            return when.reduce(contexts, function(prev, context) {
+            return when.reduce(contexts, function(prev, context, idx) {
+                portsStarted.push(context); // collect ports that are started
```

Comment:
we don't need to be sure that the port is started, we need only port object so we can call stop on it